test/views/helper/PBIKViewSpec.scala

```diff
@@ -95,6 +95,75 @@ trait PBIKViewBehaviours extends PlaySpec with JsoupMatchers {
   }
 }
 
+trait PBIKNoViewBehaviours extends PlaySpec with JsoupMatchers {
```

Comment:
I don't think duplicating the behaviour trait and its methods is necessary. Rather make implicits for the view and use the same trait.

```
trait PBIKViewBehaviours extends PlaySpec with JsoupMatchers {
  def doc(implicit view: Html): Document = Jsoup.parse(view.toString())

  def pageWithTitle(titleText: String)(implicit view: Html): Unit =
    "have a static title" in {
      doc.title must include(titleText)
    }
```

Reply:
i tried with that and didnt work, it needed to have the view passed as a param

Reply:
I have it working, when i was trying to understand why. When you're free can look at it on yours.